doc/content/getting_started/installation.md

```diff
@@ -126,6 +182,12 @@ git checkout devel
 git fetch upstream
 git rebase upstream/devel
 git submodule update moose
+git submodule update tmap8
+git submodule update cardinal
+cd cardinal
+git submodule update --recursive contrib/openmc
+git submodule update --recursive contrib/DAGMC
+git submodule update contrib/moab
 ```
 
```

Comment:
```suggestion

!alert! tip title=Disabling Cardinal
Cardinal is an optional dependency of FENIX and can be disabled to speed up build times, if Cardinal
capabilities are not needed. This can be done by simply not downloading the Cardinal submodule
(i.e., not performing the `git submodule update cardinal` command as well as the following Cardinal
dependency update commands above.)
!alert-end!

```